packages/atomic/cypress/integration/no-results-v1.cypress.ts

```diff
@@ -0,0 +1,64 @@
+import {
+  buildTestUrl,
+  injectComponent,
+  setUpPage,
+} from '../utils/setupComponent';
+import {generateAliasForSearchBox} from './search-box-selectors';
+
+describe('No Results Test Suites', () => {
+  const tag = 'atomic-no-results-v1';
+  const component = (attributes = '') => `<${tag} ${attributes}></${tag}>`;
+  const searchBox = '<atomic-search-box></atomic-search-box>';
+  const wait = 1000;
+
+  it('should not be visible when there are results', () => {
+    setUpPage(component());
```

Comment:
instead of the older setup use here, can you use the `new TestFixture()...` setup? See the facets v1 for some examples